include/wil/win32_helpers.h

```diff
@@ -887,6 +887,177 @@ bool init_once(_Inout_ INIT_ONCE& initOnce, T func)
         return false;
     }
 }
+
+namespace details
+{
+    // An iterator that can be used to enumerate processes in the system.
+    // Usage:
+    // for (auto process : process_iterator(L"ProcessName.exe"))
+    // {
+    //     // Do something with process, e.g.
+    //     // wil::unique_handle hProcess(::OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, process.th32ProcessID));
+    // }
+    struct process_entry
+    {
+        DWORD ProcessID{};
+        ULONG_PTR ThreadCount{};
+        DWORD ParentProcessID{};
+        LONG BasePriority{};
+        std::wstring ProcessName;

```

Comment:
Is this already in a region guarded by "has C++ wstring and exceptions"?

Reply:
now it is :)